docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: "3.2"
+services:
+  wordcamp.test:
```

Comment:
`MIGHT` - Can we use tabs for indentation instead of spaces, to be consistent? IIRC YAML allows that.

Reply:
Unfortunately, [YAML does not allow tabs](https://yaml.org/faq.html).

Reply:
huh, ok :)

New issue: from that FAQ it sounds like maybe we should use `.yaml` instead of `.yml`, but not a big deal.

Reply:
Fixed in https://github.com/WordPress/wordcamp.org/pull/113/commits/0a485e31005fb66c47a0c5c1bfa51ad6a6ca680e